docs/Build.md

```diff
@@ -47,10 +49,9 @@ Installing `hie` is a multi-step process:
 
 1. `git submodule sync && git submodule update --init`
 2. `hoogle generate` (`hoogle>=5.0.17` to be safe)
-3. ensure that `cabal-install` is installed in the correct version
-4. `stack --stack-yaml=stack-<X>.yaml install` or `cabal new-install -w ghc-<X>`
-5. rename `hie` binary to `hie-<X>` in `$HOME/.local/bin`, where `<X>` is the GHC version used
-6. repeat step 4 and 5 for all desired GHC versions
+3. `stack --stack-yaml=stack-<X>.yaml install` or `cabal new-install -w ghc-<X>`
```

Comment:
why no `cabal-install` step anymore?

Reply:
Before #1368 cabal was always installed independently of the target. Now it is only installed by stack if you executes explicitly the target `stack-install-cabal` or executes a `cabal-` target with stack.